images/buildkit/tests/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "version" {
+  digest = var.digest
+  script = "docker run --rm $IMAGE_NAME --version"
+}
+
+data "oci_exec_test" "build" {
+  digest = var.digest
+  script = "${path.module}/02-build.sh"
```

Comment:
The `02-` prefix is a vestige of the bygone era of `monopod` crafting shell invocations where this affected ordering.  I've been encouraging folks to just drop the prefix these days.